LibraryComponents/Classes/Core/AndesMessage/Bullets/AndesBulletView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  AndesBulletView.swift
+//  AndesUI
+//
+//  Created by Martin Ezequiel Becerra on 2/25/21.
+//
+
+import UIKit
+
+protocol AndesBulletViewDelegate: class {
```

Comment:
```suggestion
protocol AndesBulletViewDelegate: AnyObject {
```

`class` esta deprecado desde Swift 4, aunque por retrocompatibilidad es un alias de `AnyObject`.

https://sarunw.com/posts/class-only-protocols-class-or-anyobject/

Reply:
Buenísimo no sabia esto. Ahi lo cambio.